packages/config/config/devices/0x0460/qnsw-001P16.json

```
@@ -29,31 +28,58 @@
 			"maxNodes": 9
 		}
 	},
+
 	"paramInformation": [
 		{
 			"#": "1",
 			"label": "Sw1 Switch Type",
-			"description": "0",
+			"description": "This parameter defines how the Device should treat the switch (which type) connected to the SW (SW1) terminal.",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 2,
 			"defaultValue": 2,
-			"unsigned": true
+			"unsigned": true,
+            "allowManualEntry": false,
+			"options": [
+				{
+					"label": "mono-stable switch type (push button)",
+					"value": 0
+				},
+				{
+					"label": "toggle switch (contact closed - ON / contact opened - OFF)",
+					"value": 1
+				},
+				{
+					"label": "toggle switch (Device changes status when switch changes status)",
+					"value": 2
+				}
+			]
 		},
 		{
 			"#": "17",
 			"label": "Restore State of O1 After Power Failure",
-			"description": "0",
+			"description": "This parameter determines if the on/off status is saved and restored for the load connected to O (O1) after a power failure.",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 1,
 			"defaultValue": 0,
-			"unsigned": true
+			"unsigned": true,
+            "allowManualEntry": false,
+			"options": [
+				{
+					"label": "Device saves last on/off status and restores it after a power failure",
+					"value": 0
+				},
+				{
+					"label": "Device does not save on/off status and does not restore it after a power failure, it remains off",
+					"value": 1
+				}
+			]
 		},
 		{
 			"#": "19",
 			"label": "Turn O1 Off Automatically With Timer",
```

In the spirit of consistency:
```suggestion
			"label": "O1: Auto-Off Timer",
```